jenkins/spark-nightly-build.sh

```diff
@@ -89,7 +91,7 @@ for buildver in "${SPARK_SHIM_VERSIONS[@]:1}"; do
 done
 
 mvn -B clean install -pl '!tools' \
-    -PsnapshotsWithDatabricks \
+    $DIST_PROFILE_OPT \
```

Comment:
this isn't pulling the databricks jars is it?

SPARK_SHIM_VERSIONS_STR=${SPARK_SHIM_VERSIONS_STR:-"301 302 303 304 311 311cdh 312 313 320 321 330"}

I don't see DB in there and the nightly needs to publish the jar with databricks